tests/unit/InvoiceTest.php

```
@@ -0,0 +1,223 @@
+     * Test invoice with accepted payment methods.
+     *
+     * @return void
+     */
+    public function test_invoice_with_accepted_payment_methods(): void
+    {
+        $order = factory(Order::class)->create();
+        $invoice = factory(Constants::INVOICE_NAMESPACE)->create([
+            'order_id' => $order->id,
+        ]);
+
+        $methods = factory(Constants::INVOICE_ACCEPTED_PAYMENT_METHODS_NAMESPACE)->create([
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$methods'. (undefined)

(UnusedLocalVariable)